hadoop-ozone/integration-test/src/test/java/org/apache/hadoop/fs/ozone/TestOzoneFileSystem.java

```diff
@@ -154,21 +155,21 @@ public void testCreateFileShouldCheckExistenceOfDirWithSameName()
     } catch (FileAlreadyExistsException fae) {
       // ignore as its expected
     }
+
+    // Cleanup
+    fs.delete(new Path("/d1/"), true);
```

Comment:
Shouldn't the cleanup be in a `finally` block? post `mkdir` if the test fails the directory won't get cleaned up. Wouldn't that affect other tests?

Reply:
Thanks for taking a look at this @ayushtkn .

In this case it doesn't matter IMO. Cause if this test (`testCreateFileShouldCheckExistenceOfDirWithSameName`) fails and throws, `testFileSystem` as a whole will fail at this point thus the consecutive tests won't be executed.

Reply:
hmm, correct the following tests will fail. Do you think, thats a good behaviour, Ideally only the affected test should fail. If there is a common cluster, that could have been initialised in a `BeforeClass` and all other tests could have ran independently.
Anyway not related to what you are doing here. Thanx for confirming.

Reply:
Yes, I also think `@BeforeClass` is better. Maybe another jira to refactor the test class this way.

Reply:
Filed https://issues.apache.org/jira/browse/HDDS-4162